### PR TITLE
[pom repo] a couple more tests

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
@bjhargrave reverting to ask for LGTM on every PR.

File: biz.aQute.repository/test/aQute/bnd/repository/maven/pom/provider/PomRepositoryTest.java [hunk 1]
```diff
@@ -94,14 +94,51 @@ public void testBndPomRepoFile() throws Exception {
...
+		config.put("pom", "testdata/pomrepo/missing-parent.xml");
+		config.put("snapshotUrls", "https://repo1.maven.org/maven2/");
+		config.put("releaseUrls", "https://repo1.maven.org/maven2/");
+		config.put("name", "test");
+		bpr.setProperties(config);
+
+		try {
+			bpr.list(null);
+			fail("Should throw IllegalArgumentException on missing parent pom.");
+		} catch (IllegalArgumentException iae) {}
```
Comment (bjhargrave) [inline biz.aQute.repository/test/aQute/bnd/repository/maven/pom/provider/PomRepositoryTest.java]: Perhaps IllegalArgumentException is too specific? Maybe just `catch (Exception)` would be better. I would also put a comment in the catch block to indicate an exception is expected behavior.

**Comment by rotty3000** [comment]
@bjhargrave please see changes.

**Comment by bjhargrave** [comment]
LGTM

**Pull request merged**